yamux/src/session.rs

```diff
@@ -842,4 +839,50 @@ mod test {
             )
         })
     }
+
+    #[test]
+    fn test_close_session_on_stream_opened() {
```

Comment:
👍

Reply:
1. Please add a link to the issue in this test case so we know the reason why the test case is added when reading the source code.
2. There's no assertions in this test case, please add a comment to explain how the test case fails before the fixing